cli/src/commands/diff.rs

```diff
@@ -23,6 +23,10 @@ use crate::ui::Ui;
 #[derive(clap::Args, Clone, Debug)]
 pub(crate) struct DiffArgs {
     /// Show changes in this revision, compared to its parent(s)
+    ///
+    /// If the revision is a merge commit, this shows changes between (*from*)
+    /// the automatic merge of the contents of all of its parents and (*to*)
+    /// the contents of the merge commit itself.
```

Comment:
nit: better to say "the commit itself" (i.e. remove "merge")? I feel "the merge" here is a bit misleading since we have described "automatic merge" before.

Reply:
I said "the revision itself".